parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -431,6 +439,10 @@ function BasePrep {
     # Turn off Firewall to enable pods to talk to service endpoints. (Kubelet should eventually do this)
     netsh advfirewall set allprofiles state off
 
+    if ($global:EnableWindowsCilium) {
+        Enable-WindowsCiliumNetworking
```

Comment:
Again, there's a chance that this function (enable-windowsciliumnetworking) won't exist. Somewhere else in this file there's a check that a function exists before calling it. You might need to copy that.

Reply:
Now checking the command is available before invoking it.